typespec/Applications.Core/examples/2023-10-01-preview/Containers_Update.json

```diff
@@ -9,7 +9,7 @@
       "properties": {
         "application": "/planes/radius/local/resourceGroups/testGroup/providers/Applications.Core/applications/app0",
         "container": {
-          "image": "radius.azurecr.io/webapptutorial-todoapp",
+          "image": "ghcr.io/radius-project/webapptutorial-todoapp",
```

Comment:
I found some more instances of Azure Container Registry that need to be fixed:-
![Screenshot 2023-10-17 at 10 49 54 AM](https://github.com/radius-project/radius/assets/28875764/9a5ee51d-da20-4858-ba00-8d46dc7672cd)